jets_bench/src/main.rs

```diff
@@ -0,0 +1,119 @@
+//! Script to parse results from benchmarks folder and generate worst
+//! case results for each jets
+//!
+//! We rely on criterion directory structure in 0.4.0
+
+use std::collections::BTreeMap;
+
+use serde::Deserialize;
+
+#[derive(Debug, Deserialize)]
+pub struct MeasurementData {
+    // The timestamp of when these measurements were saved.
+    pub datetime: String,
+    // The number of iterations in each sample
+    pub iterations: Vec<f64>,
+    // The measured values from each sample
+    pub values: Vec<f64>,
+    // The average values from each sample, ie. values / iterations
+    pub avg_values: Vec<f64>,
+    // The statistical estimates from this run
+    pub estimates: Estimates,
+}
+
+#[derive(Debug, Deserialize)]
+pub struct Estimates {
+    pub mean: Estimate,
+    pub median: Estimate,
+    pub median_abs_dev: Estimate,
+    pub slope: Option<Estimate>,
+    pub std_dev: Estimate,
+}
+
+#[derive(Clone, PartialEq, Deserialize, Debug)]
+pub struct Estimate {
+    /// The confidence interval for this estimate
+    pub confidence_interval: ConfidenceInterval,
+    ///
+    pub point_estimate: f64,
+    /// The standard error of this estimate
+    pub standard_error: f64,
+}
+
+#[derive(Clone, PartialEq, Deserialize, Debug)]
+pub struct ConfidenceInterval {
+    pub confidence_level: f64,
+    pub lower_bound: f64,
+    pub upper_bound: f64,
+}
+
+fn main() {
```

Comment:
In 4ff2efd471d0417419a3279b664f85d2079b1249:

This can wait for a future PR but we should definitely change this to use `anyhow` and return sensible errors rather than unwrapping everywhere.